dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/GitRepositoryDetector.kt

```diff
@@ -65,7 +65,9 @@ internal class GitRepositoryDetector(
         sourceSetRoots: List<File>
     ): List<String> {
         val files = mutableListOf<String>()
+        LOGGER.info("Listing ${sourceSetRoots}")
         sourceSetRoots.forEach { sourceSetRoot ->
+            LOGGER.info("Listing ${sourceSetRoot.absolutePath}")
```

Comment:
seems excessive, the log entry above will already contain the item logged here.

Reply:
Yeah, sorry, left in from debugging. Nice catch.